cri-o.yml

```diff
@@ -169,6 +171,22 @@
         regexp: 'overlay2\.override_kernel_check=1'
         state: present
       when: (ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS')
+    - block:
```

Comment:
Does this allow us to bring up a cluster that uses runc for all trused and untrusted workloads? If not, we should have a env variable/parameter to control that.